SPECS/python-more-itertools/python-more-itertools.spec

```diff
@@ -1,22 +1,21 @@
-
-%global _description %{expand:
-Python's itertools library is a gem - you can compose elegant solutions for
-a variety of problems with the functions it provides. In more-itertools we
-collect additional building blocks, recipes, and routines for working with
-Python iterables.}
 Summary:        More routines for operating on Python iterables, beyond itertools
 Name:           python-more-itertools
 Version:        8.13.0
-Release:        2%{?dist}
+Release:        3%{?dist}
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://github.com/erikrose/more-itertools
 Source0:        %{pypi_source more-itertools}
+%global _description %{expand:
```

Comment:
Small request: could we move this macro back to the top of the file? We try to keep them there. Or at least not in between the sources and build-/run-time dependencies. I'm guessing this was the linter's doing.;)

Reply:
Yep lol this was the linter. Undoing it and then should be ready to merge :)